randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1971,17 +1971,16 @@ Extra - room_id: [42]
   > Event - Serris
       All of the following:
           Any of the following:
-              # Serris Weapon Requirements
+              # Serris Weapon Requirements (50HP)
               Charge Beam or Missiles ≥ 10
               Missiles ≥ 5 and 20 DMG Missiles
           # Core-X Requirements
           Missiles
           Any of the following:
               # Health Requirements
-              Combat (Expert) or Normal Damage ≥ 400
-              Combat (Beginner) and Normal Damage ≥ 300
+              Combat (Advanced) or Normal Damage ≥ 300
```

Comment:
link the vid: https://youtu.be/1M1GCCa4jc0

Advanced is fine considering that a blind spot exists where you literally cant get hit.